tomcat/src/main/java/com/techcourse/config/ApplicationConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.techcourse.config;
+
+import com.techcourse.controller.LoginController;
+import com.techcourse.controller.RegisterController;
+import com.techcourse.exception.GlobalExceptionHandler;
+import java.util.List;
+import org.apache.catalina.config.DefaultTomcatConfig;
+import org.apache.catalina.controller.ExceptionHandler;
+import org.apache.catalina.controller.RequestMapping;
+
+public class ApplicationConfig extends DefaultTomcatConfig {
+
+    private static final ExceptionHandler EXCEPTION_HANDLER = new GlobalExceptionHandler();
+    private static final RequestMapping REQUEST_MAPPING = RequestMapping.of(
```

Comment:
클라이언트 입장에서 List로 또 한 번 감싸기 귀찮지 않을까요?

```suggestion
    private static final RequestMapping REQUEST_MAPPING = RequestMapping.of(
            new LoginController(),
            new RegisterController()
    );
```